packages/frontend/src/pages/settings/preferences.vue

```diff
@@ -978,6 +993,31 @@ watch([
 	suggestReload();
 });
 
+async function resetAllPreferences() {
+	const { canceled, result } = await os.actions({
+		type: 'warning',
+		title: i18n.ts.resetAllPreferences,
+		text: i18n.ts.resetAllPreferencesDescription,
+		actions: [
+			{ value: 'keepThemes', text: i18n.ts.resetAllPreferencesKeepThemes },
+			{ value: 'resetAll', text: i18n.ts.resetAllPreferencesAll, danger: true },
+		],
+	});
+	if (canceled) return;
+
+	if (result === 'keepThemes') {
+		prefer.resetAll({ keepKeys: [...themePreferenceKeys] });
+	} else {
+		prefer.resetAll();
+	}
+
+	lang.value = null;
+	fontSize.value = null;
+	useSystemFont.value = false;
+
+	os.toast(i18n.ts.preferencesReset);
```

Comment:
```suggestion
	os.success();
```

成功フィードバックは個別にメッセージを出すのではなくこれで済ませます

Reply:
いや、メッセージ出さずに

```
os.waiting();
```

を処理開始で入れて

```
unisonReload();
```

を処理終了時に呼ぶべきですね